homeassistant/components/climate/__init__.py

```diff
@@ -661,19 +735,6 @@ def supported_features(self) -> ClimateEntityFeature:
         """Return the list of supported features."""
         return self._attr_supported_features
 
-    @property
-    def supported_features_compat(self) -> ClimateEntityFeature:
```

Comment:
This moves to `__getattribute__` instead as `int´ with `ClimateEntityFeature` always results in a `ClimateEntityFeature` so we need to make the check and log before.